rr_platform/launch/camera_logitech_c615.launch

```diff
@@ -0,0 +1,29 @@
+<launch>
+  <arg name="serial"/>
+  <arg name="side"/>
+
+  <node pkg="nodelet" type="nodelet" name="camera_nodelet_uvc" output="screen"
+        args="load libuvc_camera/driver /camera_nodelet_manager">
+    <!-- Parameters used to find the camera -->
+    <param name="vendor" value="0x046d"/>
+    <param name="product" value="0x082c"/>
+    <param name="serial" value="$(arg serial)"/>
+    <param name="frame_id" value="camera_$(arg side)_optical"/>
+    <param name="camera_info_url" value="package://rr_platform/conf/camera_calibration_logitech_c615.yaml"/>
+
+    <!-- image size and type -->
+    <param name="width" value="640"/>
+    <param name="height" value="480"/>
+    <param name="video_mode" value="uncompressed"/> <!-- or yuyv/nv12/mjpeg -->
+    <param name="frame_rate" value="30"/>
+    <param name="timestamp_method" value="start"/>
+
+    <!-- image quality -->
+    <param name="auto_exposure" value="0"/>  <!-- Manual = 0, Auto = 3 -->
```

Comment:
I seem to remember some other parameter we changed that aperture priority vs. shutter priority vs something else. Feels like something we need to set here despite me not remembering what it was called right now.